ui/components/app/wallet-overview/wallet-overview.js

```diff
@@ -2,9 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+// TODO: Move this function to shared
+// eslint-disable-next-line import/no-restricted-paths
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
+import { ENVIRONMENT_TYPE_FULLSCREEN } from '../../../../shared/constants/app';
+
 const WalletOverview = ({ balance, buttons, className }) => {
+  const overviewClasses = ['wallet-overview'];
+  if (getEnvironmentType() === ENVIRONMENT_TYPE_FULLSCREEN) {
+    overviewClasses.push('wallet-overview-fullscreen');
+  }
+
   return (
-    <div className={classnames('wallet-overview', className)}>
+    <div className={classnames(...overviewClasses, className)}>
```

Comment:
Nit: and optional: we can inline the className as well, unless this is international 🙏🏻 
```
className={classnames(
      'wallet-overview',
      { 'wallet-overview-fullscreen': getEnvironmentType() === ENVIRONMENT_TYPE_FULLSCREEN },
      className
    )}
```

Reply:
Good suggestion, that is nicer

Reply:
Done: cf9adf0be7